src/headerCells/SortableHeaderCell.tsx

```diff
@@ -57,12 +53,13 @@ export default function SortableHeaderCell<R, SR>({
         direction = sortDescendingFirst ? 'DESC' : 'ASC';
         break;
     }
-    onSort(column.key, direction);
+    onSort(column.key, direction, event.ctrlKey);
   }
 
   return (
     <span className={headerSortCellClassname} onClick={onClick}>
       <span className={headerSortNameClassname}>{children}</span>
+      {sortText && <span>{priority}</span>}
       <span>{sortText}</span>
```

Comment:
This can be simplified. We also need to make sure we only pass a number `priority` when the column is sorted and we have more than one column sorted. Should be `undefined` otherwise.
```suggestion
      <span>
        {priority}
        {sortText}
      </span>
```